bridge/opencensus/bridge.go

```diff
@@ -0,0 +1,216 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package opencensus
+
+import (
+	"context"
+	"fmt"
+
+	"go.opentelemetry.io/otel/global"
+
+	octrace "go.opencensus.io/trace"
+
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/codes"
+	"go.opentelemetry.io/otel/label"
+)
+
+// NewTracer returns an implementation of the OpenCensus Tracer interface which
+// uses OpenTelemetry APIs.  Using this implementation of Tracer "upgrades"
+// libraries that use OpenCensus to OpenTelemetry to facilitate a migration.
+func NewTracer(tracer otel.Tracer) octrace.Tracer {
+	return &otelTracer{tracer: tracer}
+}
+
+type otelTracer struct {
+	tracer otel.Tracer
+}
```

Comment:
Could you add `var _ octrace.Tracer = (*otelTracer)(nil)` after the type definition?

Reply:
done